upstream-community-operators/keda/1.2.0/keda.v1.2.0.clusterserviceversion.yaml

```diff
@@ -0,0 +1,330 @@
+apiVersion: operators.coreos.com/v1alpha1
+kind: ClusterServiceVersion
+metadata:
+  name: keda.v1.2.0
+  namespace: placeholder
+  annotations:
+    alm-examples: >-
+      [{"apiVersion":"keda.k8s.io/v1alpha1","kind":"KedaController","metadata":{"name":"keda","namespace":"keda"},"spec":{"watchNamespace":"","logLevel":"info","logLevelMetrics":"0"}},{"apiVersion":"keda.k8s.io/v1alpha1","kind":"TriggerAuthentication","metadata":{"name":"example-triggerauth","namespace":"example-namespace"},"spec":{"secretTargetRef":[{"key":"example-role-key","name":"example-secret-name","parameter":"exmaple-secret-parameter"}]}},{"apiVersion":"keda.k8s.io/v1alpha1","kind":"ScaledObject","metadata":{"name":"example-scaledobject","labels":{"deploymentName":"example-deployment"}},"spec":{"scaleTargetRef":{"deploymentName":"example-deployment","containerName":"example-container"},"pollingInterval":30,"cooldownPeriod":300,"minReplicaCount":0,"maxReplicaCount":100,"triggers":[{"type":"example-trigger","metadata":{"namespace":"examle-namespace"}}]}}]
+    categories: Cloud Provider
+    certified: 'false'
+    createdAt: '2020-02-06T00:00:00.000Z'
+    description: 'Operator that provides KEDA, a Kubernetes-based event driver autoscaler'
+    containerImage: 'docker.io/kedacore/keda-olm-operator:1.2.0'
+    support: 'KEDA Community'
+    capabilities: Basic Install
+    repository: 'https://github.com/kedacore/keda-olm-operator'
+spec:
+  displayName: KEDA
+  description: >+
+    ## About the managed application
+
+    KEDA is a Kubernetes-based event driven autoscaler.  KEDA can monitor event
+    sources like Kafka, RabbitMQ, or cloud event sources and feed the metrics
+    from those sources into the Kubernetes horizontal pod autoscaler.  With
+    KEDA, you can have event driven and serverless scale of deployments within
+    any Kubernetes cluster.
+
+    ## About this Operator
+
+    The KEDA Operator deploys and manages installation of KEDA Controller in the cluster.
+    Install this operator and follow installation instructions
+    on how to install KEDA in you cluster.
+
+    ### How to install KEDA in the cluster
+
+    The installation of KEDA is triggered by the creation of `KedaController` resource. 
+
+    Only resource named `keda` in namespace `keda` will
+    trigger the installation, reconfiguration or removal of the KEDA
+    Controller resource.
+
+
+    There could be only one KEDA Controller in the cluster. 
+
+  maturity: alpha
+  version: 1.2.0
+  replaces: ''
```

Comment:
@zroubalik, please comment out or remove "replaces" property as it is the first commit. Some scripts could fail on this empty value. You will use it in the next PR and point it to the last released version which is out.

Reply:
Fixed